### ISSUE TITLE
Bug: Performance Issues and Application Crashes After Update 2.5.22.0

### ISSUE THREAD

**Issue by Lukiluc29** [files-community/Files#13194]
### Description

Since the new update 2.5.22.0, I've noticed a significant regression in performance. Initially, I thought it might be a minor performance issue with my laptop, but even after restarts and such, the problems persist. So, I redirected myself to my much more powerful desktop computer, but the same issues are present. Let me explain further. When I transfer a file, for instance, to OneDrive, there's a fifty-fifty chance that the application crashes. Similarly, when I move a file to the Recycle Bin, there's a fifty-fifty chance of a crash, as well as when I attempt to empty the Recycle Bin. Some actions appear to complete successfully, like moving a file, but then the application crashes, and after the crash, it seems as if nothing happened. For me, except for opening or navigating files, the application is unusable. To be clearer, all major actions such as restoring an item, emptying the Recycle Bin, placing an item into the Recycle Bin, moving an item, etc., cause the application to crash.

### Steps To Reproduce

...

### Requirements

Investigate and identify the source of the performance regression, potentially related to the new update.

### Files Version

2.5.22.0

### Windows Version

22H2 22621.1992

### Log File

[debug.log](https://github.com/files-community/Files/files/12330051/debug.log)

**Comment by yaira2** [comment]
@Lukiluc29 thank you for the feedback, can you fill out the steps to repro? I'm pretty sure all these issues are related to #13192.

**Comment by Lukiluc29** [comment]
> @Lukiluc29 thank you for the feedback, can you fill out the steps to repro? I'm pretty sure all these issues are related to #13192.

This issue highlights the problem during deletion, except that I am experiencing other issues as well.

**Comment by Lukiluc29** [comment]
I updated the steps to reproduce.

**Comment by Lukiluc29** [comment]
> The Git issue sounds like #12737, can you confirm if it's the same?

No, it's different. Launching a Git folder and checking the Task Manager, there's nothing that increases significantly.

**Comment by Lukiluc29** [comment]
@yaira2 Do you think it would therefore not be better to close this issue and open an issue other than Git?

**Comment by Lukiluc29** [comment]
@yaira2 So don't you think it would be better to close this topic and open an issue than for Git?

**Comment by yaira2** [comment]
It might be easier to edit this to reflect the remaining points.

**Comment by Lukiluc29** [comment]
@yaira2 I finally opened a problem
